packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/index.js

```diff
@@ -1148,23 +1144,25 @@ const FormModal = () => {
                                 className="col-12"
                                 style={{
                                   marginBottom: '1.4rem',
-                                  marginTop: -2,
+                                  lineHeight: 'normal',
                                   fontWeight: 500,
                                 }}
                                 key="divider"
                               >
-                                <FormattedMessage
-                                  id={getTrad(
-                                    'form.attribute.item.settings.name'
-                                  )}
-                                />
+                                <Label htmlFor="divider-no-for">
+                                  <FormattedMessage
+                                    id={getTrad(
+                                      'form.attribute.item.settings.name'
+                                    )}
+                                  />
+                                </Label>
                               </div>
                             );
                           }
 
                           // The spacer type is used mainly to align the icon picker...
                           if (input.type === 'spacer') {
-                            return <div key="spacer" style={{ height: 11 }} />;
+                            return <div key="spacer" style={{ height: 8 }} />;
```

Comment:
Is it possible to have more than one spacer in the form modal? (The key is not unique here)

Reply:
It is not possible

Reply:
Maybe we can use different keys, specially for the spacers

```
// The spacer type is used mainly to align the icon picker...
if (input.type === 'spacer') {
  return <div key="spacer" style={{ height: 8 }} />;
}

// The spacer type is used mainly to align the icon picker...
if (input.type === 'spacer-small') {
  return <div key="spacer" style={{ height: 4 }} />;
}

if (input.type === 'spacer-medium') {
   return <div key="spacer" style={{ height: 8 }} />;
}
```

Because I guess in our case, we never have several spacers in the same form but it's still possible to create the case

Reply:
I replace all static keys by
`key={`${index}.${i}`}`
which is already used below